### PR TITLE
Add /usr Endpoint, Update VictoryBox, Add Starfield Background, Translate to Norwegian

### PR DESCRIPTION
**Pull request by eskildsvennungsen**
**Description**:
This pull request includes the following updates to our project:

1. **New Endpoint**:
   - Added a new `/usr` endpoint in our backend to handle user-specific requests.

2. **VictoryBox Update**:
   - Updated the VictoryBox component to display daily statistics.

3. **Starfield Background**:
   - Added a Starfield background to create the effect of being in space.

4. **Translation**:
   - Translated the entire webpage to Norwegian for our Norwegian-speaking users.

**Testing**:
- Verified that the `/usr` endpoint is accessible and returns correct user data.
- Ensured that the VictoryBox now displays daily stats accurately.
- Checked the Starfield background on different screen sizes to confirm it renders correctly.
- Reviewed the Norwegian translations for accuracy.

---

> "Any sufficiently advanced technology is indistinguishable from magic." - Arthur C. Clarke

Feel free to adjust any specific details or add any additional information as needed.

### CODE REVIEW

File: client/src/components/VictoryBoxStats.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  const defaultScreen = () => {
+    return (
+      <ul className='mt-3 ml-5 max-w-md space-y-1 list-disc'>
+        <li>
+          Du var <span className='px-2 bg-sky-800 text-white rounded-md'>nr. {props.data.ammount}</span> til å gjette
+          dagens land.
+        </li>
+        <li>
+          Dette greide du på <span className='px-2 bg-sky-800 text-white rounded-md'>{props.guesses} forsøk</span>.
```
Comment (lopen) [inline client/src/components/VictoryBoxStats.jsx:21]:
```suggestion
          Dette greide du på <span className='px-2 bg-sky-800 text-white rounded-md'>{props.guesses} forsøk.</span>
```

File: client/src/components/VictoryBoxStats.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        </li>
+        <li>
+          Tilsammen har <span className='px-2 bg-sky-800 text-white rounded-md'>{props.data.totGuesses} forsøk</span>{' '}
+          blitt gjort i dag.
+        </li>
+        <li>
+          Spillere har i gjennomsnitt brukt{' '}
+          <span className='px-2 bg-sky-800 text-white rounded-md'>{props.data.avgGuesses} forsøk</span> på dette landet.
```
Comment (lopen) [inline client/src/components/VictoryBoxStats.jsx:29]: hva er greia med `{' '}` her? er det for mellomrom?

File: client/src/routes/root.jsx [hunk 3]
```diff
@@ -9,10 +9,25 @@ const Root = () => {
   const [played, setPlayed] = useState('');
 
   function getDate() {
+    const norskeMåneder = [
```
Comment (lopen) [inline client/src/routes/root.jsx:12]: Selv om siden vises på norsk så er fortsatt kodebasen engelsk, skal vi ha norske eller engelske variable navn?

```suggestion
    const norwegianMonths = [
```

File: client/src/routes/root.jsx [hunk 4]
```diff
@@ -9,10 +9,25 @@ const Root = () => {
...
+      'Oktober',
+      'November',
+      'Desember',
+    ];
+
     try {
       const today = new Date();
       const day = today.getDate();
-      const month = today.toLocaleString('default', { month: 'long' });
+      const month = norskeMåneder[today.getMonth()];
```
Comment (lopen) [inline client/src/routes/root.jsx:30]: Ref. tidligere kommentar.

```suggestion
      const month = norwegianMonths[today.getMonth()];
```

File: client/src/components/Stars.jsx
Comment (lopen) [inline client/src/components/Stars.jsx:1]: Mulig vi dreper performance, men ser fett ut :rocket:

**Review by lopen — COMMENTED** [review]
Bare et par forslag :)

File: client/src/components/VictoryBoxStats.jsx (hunk 2 above)
Comment (eskildsvennungsen) [reply client/src/components/VictoryBoxStats.jsx:29]: Ja, må være der pga span havner på ny linje

File: client/src/components/Stars.jsx
Comment (eskildsvennungsen) [reply client/src/components/Stars.jsx:1]: Kan du teste på mobil?

Comment (lopen) [reply client/src/components/Stars.jsx:1]: sjekka nå, ser bra ut - ingen performace drop som jeg kan se med en gang

LGTM

**Pull request merged**